src/addScore.js

```diff
@@ -0,0 +1,19 @@
+import { saveToApi } from './api.js';
+
+const addScore = () => {
+  const addForm = document.querySelector('.add-form');
+  const name = document.getElementById('name');
+  const score = document.getElementById('score');
+
+  addForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+    saveToApi(name.value, score.value)
+      .then((response) => {
+        if (response.result === 'Leaderboard scores are well created.') {
+          window.reload();
+        }
+      });
+  });
```

Comment:
- [ ] [OPTIONAL] As already mentioned this suggestion is OPTIONAL, your project is already approved 👏 having said that it might look nice if you clear your form after submitting it so it is clear to the user that their score is submitted.

Reply:
Fixed